packages/studio-be/src/core/app/bootstrap.ts

```diff
@@ -99,7 +99,7 @@ async function start() {
   const modules = _.uniqBy(globalConfig.modules, x => x.location)
   const enabledModules = modules.filter(m => m.enabled)
 
-  const logger = await getLogger(app.logger, 'Launcher')
+  const logger = await getLogger(app.logger, '[Studio] Launcher')
```

Comment:
All logs of the studio are already prepended by [Studio], this can be removed

Reply:
yep leftover